### ISSUE TITLE
Jenkins GKE smoke e2e test failure

### ISSUE THREAD

**Issue by mtaufen** [kubernetes/kubernetes#30012]
I'm seeing the Jenkins GKE smoke e2e tests fail on these PRs:
- https://github.com/kubernetes/kubernetes/pull/29999
  + Log: https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/29999/kubernetes-pull-build-test-e2e-gke/5267/
- https://github.com/kubernetes/kubernetes/pull/29216
  + Log: https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/29216/kubernetes-pull-build-test-e2e-gke/5263/

I think it may be related to this message in the associated build-log.txt files:
```
No account authorized through gcloud.  Please fix with:

  gcloud auth login
No account authorized through gcloud.  Please fix with:

  gcloud auth login
No account authorized through gcloud.  Please fix with:

  gcloud auth login
!!! [0803 11:01:13] Max attempts reached. Will exit.
```

@ronnielai is also seeing this issue on #29925: https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/29925/kubernetes-pull-build-test-e2e-gke/5265/

Is anyone else experiencing this?

/cc @vishh @timstclair

**Comment by gmarek** [comment]
Same here: #29996

**Comment by gmarek** [comment]
I got the second failure in the row in #29996 - assuming it's a regression and bumping to P0.

**Comment by spxtr** [comment]
Fixed in #30008, please reopen if it reoccurs.

**Comment by mtaufen** [comment]
Great, thanks for fixing!
